helmfile/charts/prometheus-alerts/templates/rules/backup-status.yaml

```diff
@@ -0,0 +1,128 @@
+    - alert: InfluxDBBackup
+      annotations:
+        description: The job daily backup job influxdb-backup have failed over 48 hours.
+        runbook_url: {{ .Values.defaultRules.runbookUrl }}alert-name-influxdbbackup
+        summary: The daily backup job influxdb-backup have failed over 48 hours.
+      expr: |-
+        (
+          (min((time()-kube_job_status_completion_time{job_name=~"influxdb-backup-[0-9]*", cluster=~".*"})/3600)) > 48
+        )
+      for: 1h
+      labels:
+        severity: critical
+    - alert: VeleroBackup
+      annotations:
+        description: The job daily backup job velero-backup have failed over 24 hours.
+        runbook_url: {{ .Values.defaultRules.runbookUrl }}alert-name-velerobackup
+        summary: The daily backup job velero-backup have failed over 24 hours.
+      expr: |-
+        (
+          (time() - velero_backup_last_successful_timestamp{schedule="velero-daily-backup", cluster=~".*"}) / 60 / 60 > 24
+        )
+      for: 1h
+      labels:
+        severity: warning
+    - alert: VeleroBackup
+      annotations:
+        description: The job daily backup job velero-backup have failed over 48 hours.
+        runbook_url: {{ .Values.defaultRules.runbookUrl }}alert-name-velerobackup
+        summary: The daily backup job velero-backup have failed over 48 hours.
+      expr: |-
+        (
+          (time() - velero_backup_last_successful_timestamp{schedule="velero-daily-backup", cluster=~".*"}) / 60 / 60 > 48
+        )
+      for: 1h
+      labels:
+        severity: critical
+    - alert: ElasticsearchBackup
+      annotations:
+        description: The job daily backup job elasticsearch-backup have failed over 12 hours.
+        runbook_url: {{ .Values.defaultRules.runbookUrl }}alert-name-elasticsearchbackup
+        summary: The daily backup job elasticsearch-backup have failed over 12 hours.
+      expr: |-
+        (
+          min((time()-kube_job_status_completion_time{job_name=~"elasticsearch-backup-.*", cluster=~".*"})/3600) > 12
+        )
+      for: 1h
+      labels:
+        severity: warning
+    - alert: ElasticsearchBackup
+      annotations:
+        description: The job daily backup job elasticsearch-backup have failed over 24 hours.
+        runbook_url: {{ .Values.defaultRules.runbookUrl }}alert-name-elasticsearchbackup
+        summary: The daily backup job elasticsearch-backup have failed over 24 hours.
+      expr: |-
+        (
+          min((time()-kube_job_status_completion_time{job_name=~"elasticsearch-backup-.*", cluster=~".*"})/3600) > 24
+        )
+      for: 1h
+      labels:
+        severity: critical
+    - alert: ElasticsearchSnap
+      annotations:
+        description: The job daily backup job {{`{{`}} $labels.repository {{`}}`}} have failed over 80 hours.
+        runbook_url: {{ .Values.defaultRules.runbookUrl }}alert-name-elasticsearchsnap
+        summary: The daily backup job elasticsearch snapshot have failed over 80 hours.
```

Comment:
Why is this alert based on 80 hours?

Reply:
It is when the numbers have turned red in the dashboards. Seems like a somewhat reasonable number.

Reply:
I think it should have the same values as the elasticsearch backup job. Unless you (or someone else) has any good reason for why it's so much larger than the rest.